src/applications/mhv-landing-page/components/CardLayout.jsx

```diff
@@ -50,7 +51,8 @@ const CardLayout = ({ data }) => {
             key={`col-${y}`}
           >
             {col.title === HEALTH_TOOL_HEADINGS.MEDICAL_RECORDS &&
-            mhvTransitionalMedicalRecordsLandingPage ? (
+            mhvTransitionalMedicalRecordsLandingPage &&
```

Comment:
Note: The temp MR card only displays when the temp MR toggle is on and will not display when the phase 1 toggle is on.

Reply:
We'll see the launch version of the card if both `mhvTransitionalMedicalRecordsLandingPage` and `mhvIntegrationMedicalRecordsToPhase1` are `false`, which is a little confusing since it seems like there's no third option…

However, if both toggles are `true`, then the link to the new MR app is on the landing page but the `/my-health/records` page is still viewable.

Given at least one feature toggle will be release in percentages, it makes sense to have them both. Just needed to think through that.

Reply:
Actually, how about we just delete the transitional toggle? It is at 100% already

Reply:
I removed the transitional toggle from the landing page.